packages/eslint-plugin-react-hooks/README.md

```diff
@@ -67,7 +67,7 @@ If you want more fine-grained configuration, you can instead add a snippet like
 #### Flat Config (eslint.config.js)
 
 ```js
-import reactHooks from 'eslint-plugin-react-hooks';
+import * as reactHooks from 'eslint-plugin-react-hooks';
```

Comment:
These are functionally equal as far as I'm awarre. Why change to the (imho) uglier `* as` form?

Reply:
For use in TypeScript, since the plugin doesn't have a default `export` (for backwards compatibility), if a user doesn't have `allowSyntheticDefaultImports` turned on in their `tsconfig`, they'll get a type error unless they use this form.  Figured it would be easier to provide the form that work for both js and ts, rather than have a specific example just for TS.

Reply:
Hmm, I guess any sane user runs Typescript with `allowSyntheticDefaultImports` these days, likely coming from `moduleResultion: "bundler"`.